### ISSUE TITLE
NTR: Long-chain 3-oxoacyl-CoA thiolase activity low in fibroblasts (Lab: pathology: biopsies & morphology)

### ISSUE THREAD

**Issue by pnrobinson** [obophenotype/human-phenotype-ontology#9103]
New term request Long-chain 3-oxoacyl-CoA thiolase activity low in fibroblasts (segal_071121192022):   UMLS: 2675272

**Comment by pnrobinson** [comment]
@MickeySegal
Not sure what enzyme this is. Do we mean
this:
https://www.uniprot.org/uniprotkb/Q8NCW8/entry
or
Human peroxisomal 3-oxoacyl-coenzyme A thiolase deficiency (https://pubmed.ncbi.nlm.nih.gov/2882519/)
or ?

**Comment by MickeySegal** [comment]
The paper is from 1987, and it is unclear how it maps onto current understanding of disease, so it might be better to go by https://www.omim.org/entry/609015 which refers to "long-chain 3-oxoacyl-CoA thiolase" and is the origin of our finding name.

**Comment by pnrobinson** [comment]
see https://pubmed.ncbi.nlm.nih.gov/35782614/

**Comment by MickeySegal** [comment]
We are both referring to the same condition.  I added "long-chain 3-keto-acyl CoA thiolase (LCKAT) deficiency" as a synonym to our entry for https://omim.org/entry/609015.

**Comment by pnrobinson** [comment]
I am coding this as https://enzyme.expasy.org/EC/2.3.1.16, with accepted name acetyl-CoA C-acyltransferase, synonyms 3-ketoacyl-CoA thiolase, beta-ketothiolase reaction  acetyl-CoA + an acyl-CoA = a 3-oxoacyl-CoA + CoA

**Comment by pnrobinson** [comment]
@MickeySegal 
Adding new term: 
- ID: HP:6001024
- label: Diminished acetyl-CoA C-acyltransferase activity in cultured fibroblasts
- parents: [Abnormal enzyme activity in cultured fibroblasts (HP:4000195)](https://hpo.jax.org/app/browse/term/HP:4000195)
- pmid: 
- def: Concentration or activity of acetyl-CoA C-acyltransferase (EC 2.3.1.16) as measured in cultured fibroblasts is below the limits of normal. This enzyme catalyzes the reaction acetyl-CoA + an acyl-CoA = a 3-oxoacyl-CoA + CoA. It is also known as 3-ketoacyl-CoA thiolase and beta-ketothiolase.
- comment: 
- orcid: orcid:0000-0002-0736-9199
- github ID: https://github.com/obophenotype/human-phenotype-ontology/issues/9103
- synonyms: Reduced 3-ketoacyl-CoA thiolase activity in cultured fibroblasts(NONE)
